cpp/Test.cpp

```diff
@@ -1,72 +1,80 @@
 #include "Input.h"
 #include "Model.h"
 #include <opencv2/opencv.hpp>
+#include <assert.h>
 
-#define TILES_DIR "../../../tiles/spirals"
+//#define TILES_DIR "../../../tiles/spirals"
```

Comment:
Just delete these comments.

I couldn't comment down below so I'll do it here. On line 86 the TILES_DIR macro is still being referenced. This obviously won't work anymore.